pkg/ui/workspaces/cluster-ui/src/databases/util.tsx

```diff
@@ -58,31 +58,71 @@ export class GetDatabaseInfoError extends Error {
   }
 }
 
-// getNodesByRegionString converts a list of node ids and map of
-// node ids to region to a string of node ids by region, ordered
-// by region name, e.g.
-// regionA(n1, n2), regionB(n2,n3), ...
+/** Store ids and node ids are both of type `number[]`. To disambiguate, a
+ * `kind` field is included in the type. */
+export type Stores = { kind: "store", ids: number[] };
```

Comment:
Expanding the type is the only way I could think to disambiguate the data of type `number[]`... Open to alternatives here.